halo-ontology/.htaccess

```diff
@@ -0,0 +1,31 @@
+# Turn off MultiViews
+Options -MultiViews
+
+# Directive to ensure .owl files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .owl
+
+# Rewrite engine setup
+RewriteEngine On
+
+# Base directory
+RewriteBase /halo-ontology/
+
+# Rewrite rule to serve HTML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^$ https://github.com/navapatn/halo-ontology/blob/main/ontology/index-en.html [R=303,L]
```

Comment:
this rewrite rule will serve a page in github. You can serve it with github pages easily, which is the target doc you are aiming for.